README.md

```diff
@@ -140,27 +178,66 @@ _After_ you have ***contributed*** to the project,
 _e.g: by commenting on or creating an issue_, <br />
 -->
 If you've found the project interesting, helpful or useful in _any way_,
-_please_ share the love by starring it! <br />
+_please_ ***star*** the repository on GitHub! <br />
 Stars show _other_ members in the developer community
 that it's a _**worthwhile project** or **learning resource**_
 and one that can offer value to other developers like you! 🌟
 
 
 <br />
 
-## Questions?
+# Questions?
```

Comment:
Good question. I'm trying to present the content in the order that a ***brand new*** person will "_need it_"... but I'm not at _all_ "_precious_" about it. I was just frustrated with the pre-images version of the doc and needed to make it more friendly and precise...

Happy for any changes to be made.